fast/stages/2-networking-a-simple/spoke-ncc.tf

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2022 Google LLC
```

Comment:
nit: I'd take the chance to update the copyrights of all files we touch

Reply:
I already had another commit lined up, but didn't want to ruin the party for you. 
Coming in :)